gammapy/data/ivoa.py

```diff
@@ -23,217 +31,224 @@ def obscore_def():
         unit="s",
         description="Total exposure time",
         dtype="f8",
         meta={"Utype": "Char.TimeAxis.Coverage.Support.Extent"},
     )
-    obscore_default[20] = Column(
+    obscore_table[20] = Column(
         name="t_resolution",
         unit="s",
         description="Temporal resolution FWHM",
         dtype="f8",
         meta={"Utype": "Char.TimeAxis.Resolution.Refval.valueResolution.Refval.value"},
     )
-    obscore_default[21] = Column(
+    obscore_table[21] = Column(
         name="em_xel",
         unit="",
         description="Number of elements along the spectral axis",
         dtype="i4",
         meta={"Utype": "Char.SpectralAxis. numBins"},
     )
-    obscore_default[22] = Column(
+    obscore_table[22] = Column(
         name="em_min",
-        unit="m",
+        unit="TeV",
         description="start in spectral coordinates",
         dtype="f8",
         meta={"Utype": "Char.SpectralAxis.Coverage.Bounds.Limits.LoLimit"},
     )
-    obscore_default[23] = Column(
+    obscore_table[23] = Column(
         name="em_max",
-        unit="m",
+        unit="TeV",
         description="stop in spectral coordinates",
         dtype="f8",
         meta={"Utype": "Char.SpectralAxis.Coverage.Bounds.Limits.HiLimit"},
     )
-    obscore_default[24] = Column(
+    obscore_table[24] = Column(
         name="em_res_power",
         unit="",
         description="Value of the resolving power along the spectral axis(R)",
         dtype="f8",
         meta={"Utype": "Char.SpectralAxis.Resolution.ResolPower.refVal"},
     )
-    obscore_default[25] = Column(
+    obscore_table[25] = Column(
         name="o_ucd",
         unit="",
         description="Nature of the observable axis",
         dtype="U30",
         meta={"Utype": "Char.ObservableAxis.ucd"},
     )
-    obscore_default[26] = Column(
+    obscore_table[26] = Column(
         name="pol_xel",
         unit="",
         description="Number of elements along the polarization axis",
         dtype="i4",
         meta={"Utype": "Char.PolarizationAxis.numBins"},
     )
-    obscore_default[27] = Column(
+    obscore_table[27] = Column(
         name="facility_name",
         unit="",
         description="The name of the facility, telescope space craft used for the observation",
         dtype="U10",
         meta={"Utype": "Provenance.ObsConfig.Facility.name"},
     )
-    obscore_default[28] = Column(
+    obscore_table[28] = Column(
         name="instrument_name",
         unit="",
         description="The name of the instrument used for the observation",
         dtype="U25",
         meta={"Utype": "Provenance.ObsConfig.Instrument.name"},
     )
     tab_default = Table()
-    for var in obscore_default:
+    for var in obscore_table:
         tab_default.add_column(var)
     return tab_default
 
 
-def _obscore_row(base_dir, single_obsID, obs_publisher_did, access_url, table):
+def _obscore_row(
+    base_dir,
+    single_obsID,
+    obs_publisher_did,
```

Comment:
Only the parameter `single_obsID' is described in the docstring. Even if this function is not exposed, the description of all parameters and their default value is important for the other developers.